ci-operator/step-registry/telcov10n/functional/ocp-baremetal-ipi-deployment/OWNERS

```diff
@@ -0,0 +1,5 @@
+approvers:
+- natifridman
+- shaior
+- ccardenosa
+- cplacani
```

Comment:
You should add your user here, like in the other OWNERS files you created.

Reply:
added

Reply:
Hi, did you push the changes? because I still don't see it here.